cg/services/order_validation_service/models/case.py

```diff
@@ -7,11 +7,11 @@
 
 
 class Case(BaseModel):
-    data_delivery: DataDelivery
+    data_delivery: DataDelivery | None = None
     internal_id: str | None = None
-    name: str = Field(pattern=NAME_PATTERN, min_length=2, max_length=128)
-    priority: PriorityTerms = PriorityTerms.STANDARD
-    samples: list[Sample]
+    name: str | None = Field(default=None, pattern=NAME_PATTERN, min_length=2, max_length=128)
+    priority: PriorityTerms | None = None
```

Comment:
Depends on what we want but currently priority is required in the Frontend but not used in the backend for existing cases